src/common/redis_module/chain_module.h

```diff
@@ -0,0 +1,50 @@
+#ifndef RAY_CHAIN_MODULE_H_
+#define RAY_CHAIN_MODULE_H_
+
+#include <functional>
+
+#include "redismodule.h"
+
+// NOTE(zongheng): this duplicated declaration serves as forward-declaration
+// only.  The implementation is supposed to be linked in from credis.  In
```

Comment:
Can you specify where exactly the implementation is in the comment? I'm having trouble finding it in the credis repo.